react-components/src/components/Toolbar/Image360HistoricalOverviewToolbar.tsx

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Detail, Flex } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+import { Thumbnail } from '../utils/Thumbnail';
+
+export interface Image360RevisionDetails{
+  revisionDate?: string;
+  revisionImageUrl?: string;
+};
+
+export interface Image360HistoricalOverviewToolbarProps{
+  stationId?: string;
+  stationName?: string;
+  collectionId?: string;
+  revisionCollection?: Image360RevisionDetails[];
+};
+
+export const Image360HistoricalOverviewToolbar = ({stationId, stationName, collectionId, revisionCollection}: Image360HistoricalOverviewToolbarProps) => {
+  return(
+    <OverviewContainer>
+
+      <StyledSubFlex direction='column'>
+        <StyledFlex>{stationName}</StyledFlex>
+        <StyledDetail>Station: {stationId}</StyledDetail>
+        <StyledDetail>Collection: {collectionId}</StyledDetail>
+      </StyledSubFlex>
+
+    <StyledLayoutGrid>
+        { revisionCollection?.map((revisionDetails) => (
+          <RevisionItem onClick={()=>{alert(revisionDetails.revisionDate + ' is selected')}}>
+            <Thumbnail imageUrl={revisionDetails.revisionImageUrl} isLoading={false}/>
+            <Detail>{revisionDetails.revisionDate}</Detail>
+          </RevisionItem>
+        ))
+        }
+    </StyledLayoutGrid>
+
+    </OverviewContainer>
+  )
```

Comment:
I see there is some misalignment in the elements here :sweat_smile: Don't need to fix now, but we should probably have a `prettier` and/or `eslint` configuration for this directory as well